docs/src/Box.doc.js

```diff
@@ -480,143 +540,235 @@ card(
-  `}
-    id="border"
-    name="Borders"
-  >
-    <Combination
-      id="border"
-      borderStyle={['sm', 'lg', 'shadow']}
-      showHeading={false}
-      layout="4column"
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`
+      Auto margin is a useful tool when using flexbox layouts. When a flex container has extra space and no components are set to grow, the browser would normally place elements next to each other. By setting any of the margin properties to "auto", the margin will extend to fill the extra space.
+
+      This can be seen in our Callout component, where the actions container uses \`marginStart="auto"\` to automatically adjust the actions at smaller screen sizes.
+      `}
+      title="Auto Margins"
     >
-      {(props) => <Box width={60} height={60} rounding="circle" color="white" {...props} />}
-    </Combination>
-  </Card>,
-);
-
-card(
-  <Combination
-    id="color"
-    name="Colors"
-    layout="4column"
-    color={[
-      'red',
-      'white',
-      'lightGray',
-      'gray',
-      'darkGray',
-      'green',
-      'pine',
-      'olive',
-      'blue',
-      'navy',
-      'midnight',
-      'purple',
-      'orchid',
-      'eggplant',
-      'maroon',
-      'watermelon',
-      'orange',
-      'transparent',
-      'transparentDarkGray',
-      'lightWash',
-      'darkWash',
-    ]}
-  >
-    {(props) => <Box width={60} height={60} rounding="circle" {...props} />}
-  </Combination>,
-);
+      <MainSection.Card
+        cardSize="lg"
+        defaultCode={`
+<Box marginStart={12} marginEnd={12}>
+  <Callout
+    type="info"
+    iconAccessibilityLabel="Info icon"
+    title="Your business account was created!"
+    message="Apply to the Verified Merchant Program!"
+    primaryAction={{href: "https://pinterest.com", label:"Get started"}}
+    secondaryAction={{href: "https://pinterest.com", label:"Learn more"}}
+    dismissButton={{
+      accessibilityLabel: 'Dismiss banner',
+      onDismiss: ()=>{},
+    }}
+  />
+</Box>
+`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection
+      description={`
+    Position is static by default but can be made absolute. \`Box\` has helpers to help align to absolute edges (top, bottom, left, right). These can be used in combination with padding to achieve desired offsets from edges.
 
-card(
-  <Card
-    description={`
-    Sets a border radius for the Box. Options are "circle" or "pill" for fully rounded corners or 0-8 representing the radius in 4px increments.
-  `}
-    id="rounding"
-    name="Rounding"
-  >
-    <Combination
-      id="rounding"
-      rounding={['pill', 'circle', 0, 1, 2, 3, 4, 5, 6, 7, 8]}
-      showHeading={false}
+    The SearchField uses absolute positioning to position the search Icon at the beginning of the input.
```

Comment:
Same question here: it seems like we're trying to teach people how to create their own custom versions of our components